src/content/en/2020/media.md

```diff
@@ -4,13 +4,14 @@ part_number: I
 chapter_number: 5
 title: Media
 description: Media chapter of the 2020 Web Almanac covering image file sizes and formats, responsive images, client hints, lazy loading, accessibility and video.
-authors: [tpiros, bseymour]
-reviewers: [nhoizey, zcorpan, colinbendell, dougsillars]
+authors: [tpiros, bseymour, eeeps]
+reviewers: [nhoizey, colinbendell, dougsillars]
 analysts: [smatei]
-editors: []
+editors: [bazzadp]
 translators: []
-#tpiros_bio: TODO
-#bseymour_bio: TODO
+tpiros_bio: Tamas Piros is a Developer Evangelist, Google Developer Expert and Technical Trainer.
+bseymour_bio: Ben Seymour is a Dynamic Media & Content Specialist at <a href="https://cloudinary.com/">Cloudinary</a>.
+eeeps_bio: Eric Portis is a Web Platform advocate at <a href="https://cloudinary.com/">Cloudinary</a>.
```

Comment:
@tpiros , @bseymour , @eeeps I've added a basic bio here. Please update as you see fit.

Reply:
I have updated mine.

Reply:
@eeeps please make your update as well so that we close this 🥳